lib/CArtHandler.h

```diff
@@ -375,7 +372,7 @@ class DLL_LINKAGE CArtifactFittingSet : public CArtifactSet
 {
 public:
 	CArtifactFittingSet(ArtBearer::ArtBearer Bearer);
-	void setNewArtSlot(ArtifactPosition slot, CArtifactInstance * art, bool locked);
+	void setNewArtSlot(const ArtifactPosition & slot, CArtifactInstance * art, bool locked);

```

Comment:
setNewArtSlot is also "override". could you please add this keyword? And why are the methods: CArtifactFittingSet::setNewArtSlot, CArtifactSet::setNewArtSlot fixed differently?

Reply:
It is not fixed differently, it is identical AFAIK. What are you mean?
```c++
void CArtifactFittingSet::setNewArtSlot(const ArtifactPosition & slot, CArtifactInstance * art, bool locked)
```

```c++
void CArtifactSet::setNewArtSlot(const ArtifactPosition & slot, CArtifactInstance * art, bool locked)
```

Reply:
@SoundSSGood method from child class removed.